dev-portal/src/pages/Dashboard.jsx

```diff
@@ -97,6 +127,13 @@ export default observer(() => {
                 firstApiName
               )}
             </Header>
+            {usagePlan.throttle && (
+              <Message info>
+              <p>
+                Requests limited to {usagePlan.throttle.rateLimit} per second, and {usagePlan.throttle.burstLimit} in a burst.
```

Comment:
Customers sometimes have a hard time grok'ing this. We should link them to whatever documentation we have in the runbook for questions about token bucket (I think it's [just wikipedia](https://en.wikipedia.org/wiki/Token_bucket)).